system/src/system_power_manager.cpp

```diff
@@ -176,14 +190,16 @@ void PowerManager::handleUpdate() {
   // consecutively. The 1st reads fault register status from the last read and the 2nd
   // reads the current fault register status.
   volatile uint8_t lastFault = power.getFault();
-  (void)lastFault;
-  const uint8_t curFault = power.getFault();
+  const uint8_t curFault = power.getFault() | lastFault;
```

Comment:
We didn't `|` the `lastFault`, any reason why we do it now?

Reply:
Correct